flocker/restapi/testtools.py

```diff
@@ -211,6 +211,42 @@ def setUp(self):
     MemoryTests.__module__ = mixinClass.__module__
     return RealTests, MemoryTests
 
+
+def buildUNIXIntegrationTests(mixinClass, name, fixture):
+    """
+    Build L{TestCase} class that runs the tests in the mixin class with
+    real queries over a UNIX socket.
+
+    @param mixinClass: A mixin class for L{TestCase} that relies on having a
```

Comment:
Oh wait ... maybe?  Or does the plugin actually use epytext everywhere?  Whaaat?